changelog/28126.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:improvement
+auth/cert: support watching changes on certificate/key files and notifying the auth handler when `enable_reauth_on_new_credentials` is enabled.
+website/docs: add docs for the new option `reload_period` for auth/cert and `enable_reauth_on_new_credentials` for Auto-Auth.
```

Comment:
```suggestion
```

Reply:
No need to add a changelog for docs -- changelogs are for released versions of Vault.